app.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express');
-const { getAllTopics, getHealthCheck } = require('./controllers/controllers');
+const { getHealthCheck, getEndpoints } = require('./controllers/app.controllers');
+const { getAllTopics } = require('./controllers/topics.controllers')
 const app = express();
 
 app.use(express.json());
```

Comment:
I spy a line of code that isn't being utilised in your API at this point 👀 
`app.use(express.json());`